internal/server/stat_point.go

```diff
@@ -130,6 +135,13 @@ func (s *Server) GetStatSetWithinPlace(
 	childType := in.GetChildType()
 	date := in.GetDate()
 
+	log.Printf(
```

Comment:
would be nice to get this type of logging for all requests

Reply:
+1

Reply:
Sg. I will do a PR for all requests